src/mmw/js/src/user/models.js

```diff
@@ -60,6 +60,14 @@ var UserModel = Backbone.Model.extend({
 var UserProfileModel = Backbone.Model.extend({
     defaults: {
         was_skipped: false,
```

Comment:
Is this `was_skipped` still used anywhere?

Reply:
Addressed in fixup db7e61d